### PR TITLE
scripts: add script to migrate testruns between projects

### PR DESCRIPTION
**Pull request by mwasilew**
In case there is a need to move test runs from one project to the other
the script can move them basing on matching environment. Example call:
./manage.py runscript migrate_test_runs --script-args \
  env_slug=<environment slug> \
  old_project_slug=<from project slug> \
  new_project_slug=<to project slug>  --traceback

Signed-off-by: Milosz Wasilewski <milosz.wasilewski@linaro.org>

### CODE REVIEW

**Comment by mwasilew** [comment]
This feature was requested to split test runs from one project to two separate ones.

File: scripts/migrate_test_runs.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    for build in old_project.builds.all():
+        if build.test_runs.filter(environment=env):
+            print("moving build: %s" % build)
+            new_build, _ = Build.objects.get_or_create(
+                version=build.version,
+                project=new_project,
+                created_at=build.created_at)
+            for testrun in build.test_runs.filter(environment=env):
+                testrun.build = new_build
```
Comment (terceiro) [inline scripts/migrate_test_runs.py]: do we need to care about the case in which there are no test runs left in the original build?

File: scripts/migrate_test_runs.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    if 'old_project_slug' not in parsed_args.keys():
+        print("ERROR: old_project_slug missing")
+        sys.exit(1)
+    if 'new_project_slug' not in parsed_args.keys():
+        print("ERROR: new_project_slug missing")
+        sys.exit(1)
+    if 'env_slug' not in parsed_args.keys():
+        print("ERROR: env_slug missing")
+        sys.exit(1)
```
Comment (terceiro) [inline scripts/migrate_test_runs.py]: in a management command this would be handled by the framework, you just need to declare the arguments

**Comment by mwasilew** [comment]
I guess you're right. I thought about mgmt command shortly after I submitted the PR. Let me change it. It makes little difference to the code and doesn't require django-extras as dependency.

**Comment by mwasilew** [comment]
I updated to mgmt command. Now ready for review.

**Comment by terceiro** [comment]
LGTM

**Pull request merged**